notebooks/official/model_monitoring/get_started_with_model_monitoring_automl.ipynb

```diff
@@ -576,18 +545,18 @@
         "\n",
         "Next, you run the created DAG to start the training job by invoking the method `run`, with the following parameters:\n",
         "\n",
-        "- `dataset`: The `Dataset` resource to train the model.\n",
+        "- `dataset`: The dataset resource to train the model.\n",
         "- `model_display_name`: The human readable name for the trained model.\n",
         "- `training_fraction_split`: The percentage of the dataset to use for training.\n",
         "- `test_fraction_split`: The percentage of the dataset to use for test (holdout data).\n",
         "- `validation_fraction_split`: The percentage of the dataset to use for validation.\n",
         "- `target_column`: The name of the column to train as the label.\n",
         "- `budget_milli_node_hours`: (optional) Maximum training time specified in unit of millihours (1000 = hour).\n",
-        "- `disable_early_stopping`: If `True`, training maybe completed before using the entire budget if the service believes it cannot further improve on the model objective measurements.\n",
+        "- `disable_early_stopping`: If `False`, training maybe completed before using the entire budget if the service believes it cannot further improve on the model objective measurements.\n",
```

Comment:
change "maybe" to "may be" and "believes" to "detects that" and "cannot" to "can't". Delete "on"

- If `False`, training may be completed before using the entire budget if the service detects that it can't further improve the model objective measurements